framework/doc/content/source/auxkernels/AdvectiveFluxAux.md

```diff
@@ -0,0 +1,27 @@
+# AdvectiveFluxAux
+
+!syntax description /AuxKernels/AdvectiveFluxAux
+
+## Description
+
+The `AdvectiveFluxAux` AuxKernel is used to compute the component of the flux vector for advection problems. The flux is computed as
```

Comment:
```suggestion
The `AdvectiveFluxAux` AuxKernel is used to compute a component of an advective flux vector. The flux is computed as
```
there could be other fluxes at hand in an advection-diffusion problem for example